packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -159,14 +177,18 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
           </Text>
         </Label>
         {hasTrimMode && (
-          <TrimButton
-            variant={BUTTON_VARIANTS.RECTANGLE}
-            type={BUTTON_TYPES.SECONDARY}
-            size={BUTTON_SIZES.SMALL}
-            onClick={toggleTrimMode}
-          >
-            {__('Trim', 'web-stories')}
-          </TrimButton>
+          <TrimWrapper>
+            <TrimButton
+              disabled={isTrimming}
```

Comment:
@spacedmonkey I saw that the Mute button is disabled when muting but it's hidden when trimming or transcoding. I made it similar for consistency, however, it would make more sense to me if both buttons would still be visible when transcoding/trimming/muting but just disabled. Thoughts?